test/backend/db_tests.js

```diff
@@ -695,11 +758,20 @@ module.exports = function(config, DB) {
               })
               .then(function() {
                 t.pass('.createAccountResetToken() did not error')
-                return db.resetAccount(ACCOUNT.uid, ACCOUNT)
+                // lock the account to ensure the unlockCode is deleted
+                return db.lockAccount(uid, { lockedAt: lockedAt, unlockCode: unlockCode })
+              })
+              .then(function() {
+                t.pass('.lockAccount() did not error')
+                return db.unlockCode(uid)
+              })
+              .then(function(unlockCode) {
+                t.ok(unlockCode.unlockCode)
```

Comment:
Nit: add a comment string so we know what the test is here (not just checking against null, but checking the unlockCode). I know you hate me. :-P